superset/config.py

```diff
@@ -563,6 +563,8 @@ class D3TimeFormat(TypedDict, total=False):
     # If on, you'll want to add "https://avatars.slack-edge.com" to the list of allowed
     # domains in your TALISMAN_CONFIG
     "SLACK_ENABLE_AVATARS": False,
+    # Allow users to optionally specify date formats in email subjects, which will be parsed if enabled.
+    "DATE_FORMAT_IN_EMAIL_SUBJECT": False,
```

Comment:
Can you mention this new feature flag in UPDATING.md

Reply:
thanks for reviewing! i have updated